tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -209,21 +209,12 @@ private void serveRunnningNormallyPage(
 
     private void includeCss(HttpServletRequest request, PrintWriter out) {
         final String contextPath = request.getContextPath();
-        final String cb = getCacheBustingExtension(request);
-        out.write(
-                "<link rel='stylesheet' href='" + contextPath + "/surveytool" + cb + ".css' />\n");
```

Comment:
This wasn't working properly, tended to cache the out of date file.  Webpack worked correctly.

Reply:
It is used when loading the error and waiting pages still, since the bundle may not be loaded properly.